tensorboard/webapp/widgets/line_chart_v2/sub_view/line_chart_axis_view.ts

```diff
@@ -56,6 +62,40 @@ const DAY_IN_MS = 24 * 1000 * 60 * 60;
         </g>
       </svg>
     </div>
+    <mat-menu
+      #manualControl="matMenu"
+      xPosition="before"
+      [yPosition]="axis === 'y' ? 'above' : 'below'"
+    >
+      <div
+        class="extent-edit-input"
+        (click)="$event.stopPropagation()"
+        (keydown)="keydownPreventClose($event)"
+      >
+        <label>min</label>
+        <input #minInput tabindex="1" type="number" [value]="axisExtent[0]" />
```

Comment:
When manually testing these inputs:
- I open the menu, the min input shows X (current min)
- I edit the min value to X2, then I dismiss the menu
- I reopen the menu

Actual: input still shows X2, even though current value is X
Expected: input should show X, because I expect the form to reflect the current state of the chart